opentelekomcloud/provider.go

```diff
@@ -250,6 +250,7 @@ func Provider() terraform.ResourceProvider {
 			"opentelekomcloud_identity_group_v3":                  resourceIdentityGroupV3(),
 			"opentelekomcloud_identity_group_membership_v3":       resourceIdentityGroupMembershipV3(),
 			"opentelekomcloud_images_image_v2":                    resourceImagesImageV2(),
+			"opentelekomcloud_images_ims_v2":                      resourceImagesImsV2(),
```

Comment:
how about naming it as opentelekomcloud_ims_image_v2, just like ecs_instance_v1, evs_volume_v3 , SERVICE_RESOURCE pattern.

Reply:
yep, it looks better and I will rename the resource to opentelekomcloud_ims_image_v2